api/src/damnit_api/filewatcher/cache.py

```diff
@@ -0,0 +1,27 @@
+import os
+import time
+from pathlib import Path
+TTL = 5.0
+MAX_AGE = 60.0
+
+file_mtime_cache: dict[str, dict[str, float]] = {}
+
+
+def get(file_path: str) -> float:
+    now = time.time()
```

Comment:
It's unlikely to ever matter here, but the clock `time.time()` uses can be adjusted back or forward if the system time changes. If you only need 'time since' within the program, not absolute time, then `time.monotonic()` gives you a clock that just keeps increasing.

```suggestion
    now = time.monotonic()
```